src/Queue/Task/MailerTask.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Queue\Queue\Task;
+
+use Cake\Log\Log;
+use Cake\Mailer\MailerAwareTrait;
+use Queue\Model\QueueException;
+use Queue\Queue\Task;
+
+/**
+ * A convenience task ready to use for asynchronously sending reusable emails via mailer classes.
+ *
+ * @author Mark Scherer
+ * @license http://www.opensource.org/licenses/mit-license.php The MIT License
+ */
+class MailerTask extends Task {
+
+	use MailerAwareTrait;
+
+	/**
+	 * @var int
+	 */
+	public $timeout = 120;
+
+	/**
+	 * @var \Cake\Mailer\Mailer
+	 */
+	public $mailer;
```

Comment:
can this be protected?
tests can use Reflection to access it if needed

Reply:
sure